packages/sdks/mitosis.config.cjs

```diff
@@ -400,6 +400,20 @@ const ANGULAR_RENAME_NG_ONINIT_TO_NG_AFTERCONTENTINIT_PLUGIN = () => ({
     post: (code) => {
       if (code?.includes('selector: "blocks-wrapper"')) {
         code = code.replace('ngOnInit', 'ngAfterContentInit');
+        const ngOnChangesIndex = code.indexOf('ngOnChanges');
+        if (ngOnChangesIndex > -1) {
+          code = code.replace(/^\s*\/\/\s*@ts-expect-error.*$/gm, '');
+          code = code.replace(
+            'ngOnChanges(changes: SimpleChanges) {',
+            // trigger a re-render of the view when blocks got updated and the new children content has been fully initialized
```

Comment:
Could we document here in more detail? A brief summary of why it's needed, essentially a tl;dr of your looms. You'll know how to phrase this in the most accurate way, but something along the lines of:

> Since the angular SDK manually handles the creation of the dynamic blocks and attaching them as children of `BlocksWrapper` in the DOM, it must also manually handle their re-renders on content change in the visual editor. `ngAfterContentChecked` runs after children were checked for changes, which is the earlier point we can safely append new blocks, otherwise they are appended outside the existing array of Blocks, therefore pushed to the top of the list.